embed/config.go

```diff
@@ -246,6 +245,9 @@ func (cfg *configYAML) configFromFile(path string) error {
 		cfg.ACUrls = []url.URL(u)
 	}
 
+	if (cfg.Durl == "" && cfg.DNSCluster == "") && cfg.InitialCluster == "" {
```

Comment:
probably have `cfg.InitialCluster == cfg.InitialClusterFromName(DefaultName)` instead of `== ""` and keep the `InitialCluster` line in `NewConfig`? otherwise the default will be blank for non-yaml cases